geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -205,8 +204,7 @@ object RichSimpleFeatureType {
 
     def getBinTrackId: Option[String] = sft.getAttributeDescriptors.find(_.isBinTrackId()).map(_.getLocalName)
 
-    def getSchemaVersion: Int =
-      userData[String](SCHEMA_VERSION_KEY).map(_.toInt).getOrElse(CURRENT_SCHEMA_VERSION)
+    def getSchemaVersion: Int = userData[String](SCHEMA_VERSION_KEY).map(_.toInt).getOrElse(10)
```

Comment:
Hmm...  the '10' here is CURRENT_SCHEMA_VERSION, no? 

I'm a bit weary of moving between a well-commented object to a magic number:).

Thoughts?

Reply:
yeah, 10 was the last ever value for CURRENT_SCHEMA_VERSION - CURRENT_SCHEMA_VERSION stopped being used back 1.3.0-ish time frame, when we switched to tracking the version of each index separately. I should probably just remove this whole method actually, or at least mark it deprecated... let me look into that.

Reply:
For now, I've commented the magic number here and marked the method as deprecated. I opened a ticket to remove it fully: https://geomesa.atlassian.net/browse/GEOMESA-2114. Does that sound ok to you?